xwiki-rendering-test/src/main/java/org/xwiki/rendering/test/cts/TestDataParser.java

```diff
@@ -151,6 +155,13 @@
                 testDataOUT.syntaxData = inheritedSyntaxData.getRight().getLeft();
                 testDataOUT.syntaxExtension = inheritedSyntaxData.getRight().getRight();
             }
+
+            try {
+                currentConfiguration = parseTestConfiguration(inheritedSyntaxDirectory, ctsRootPackageName,
+                    classLoader);
+            } catch (Exception ignored) {
+                break;
```

Comment:
Why is this ignored? Maybe add a comment to explain why.

Reply:
Good question, I think my initial assumption was somehow that we might get an exception if there is no more syntax to inherit from, but looking again this doesn't make any sense. I think this exception shouldn't be ignored actually so we notice errors with the configuration files. However, then we would need to change the `throws`-clause which would break all callers of this methods from my understanding. I've just tried doing that and it seems revapi is not complaining which really surprises me. So is it okay to change the `throws`-clause from `IOException` to `Exception`?

Reply:
> I've just tried doing that and it seems revapi is not complaining which really surprises me

We exclude test stuff AFAIK.